src/main/java/me/csed2/moneymanager/subscriptions/Subscription.java

```diff
@@ -1,20 +1,36 @@
 package me.csed2.moneymanager.subscriptions;
 
+import lombok.Getter;
 import me.csed2.moneymanager.cache.Cacheable;
 import me.csed2.moneymanager.transactions.Transaction;
 
 import java.util.Arrays;
 import java.util.Date;
 import java.time.format.DateTimeFormatter;
-
+@Getter
 public class Subscription extends Transaction implements Cacheable{
 
     private int timeCycle;
-    private Boolean cancelMe;
-    private Date startDate = date;
+    private Boolean cancelMeBool;
+    private String timeCycleUnit;
+    private String commencement;
+    private String positiveList[]={"yes","y","true","t","affirm"};
```

Comment:
in Java, String arrays are defined as String[], unlike in C which is String positiveList[]